services-core/src/economic_viability.rs

```diff
@@ -171,12 +174,12 @@ fn gas_price_cap(native_token_price: f64, earned_fee: f64, num_trades: usize) ->
 
 /// Fixed values.
 pub struct FixedEconomicViabilityComputer {
-    min_average_fee: Option<u128>,
-    max_gas_price: Option<U256>,
+    min_average_fee: u128,
+    max_gas_price: U256,
```

Comment:
I don't see a reason to ever store None and it makes the code simpler.

Reply:
Seems legit and fine by me!